applications/crossbar/src/modules/cb_cdrs.erl

```diff
@@ -35,6 +35,14 @@
 -define(MAX_BULK, whapps_config:get_integer(?MOD_CONFIG_CAT, <<"maximum_bulk">>, 50)).
 -define(CB_LIST_BY_USER, <<"cdrs/listing_by_owner">>).
 -define(CB_LIST, <<"cdrs/crossbar_listing">>).
+-define(CB_GROUP_LIST, <<"cdrs/group_listing">>).
+-define(CB_GROUP_LIST_BY_USER, <<"cdrs/group_listing_by_owner">>).
+-define(CB_GROUP_LIST_BY_ID, <<"cdrs/group_listing_by_id">>).
+
+-define(PATH_GROUP, <<"group">>).
+-define(PATH_LEGS, <<"legs">>).
+
+-define(KEY_CCV, <<"custom_channel_vars">>).
```

Comment:
I think this should be defined in `wh_types.hrl` given the amount of modules using it